packages/isolated-block-editor/src/components/block-editor/convert-to-group-buttons.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import { switchToBlockType } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { BlockSettingsMenuControls } from '@wordpress/block-editor';
+
+export function ConvertToGroupButton( {
+	onConvertToGroup,
+	onConvertFromGroup,
+	isGroupable = false,
+	isUngroupable = false,
+} ) {
+	if ( ! isGroupable && ! isUngroupable ) {
+		return null;
+	}
+
+	return (
+		<BlockSettingsMenuControls>
+			{ ( { onClose } ) => (
+				<>
+					{ isGroupable && (
+						<MenuItem
+							onClick={ () => {
+								onConvertToGroup();
+								onClose();
+							} }
+						>
+							{ _x( 'Group', 'verb' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Group', { context: 'NewDash Page Title'} )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/isolated-block-editor-package)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `NewDash Page Title`

 Would it make sense to reuse one of the above?